pkg/spi-shared/config/config_test.go

```diff
@@ -18,13 +18,15 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 )
 
 func TestRead(t *testing.T) {
+	os.Setenv(sharedSecretEnv, "secretValue")
```

Comment:
I'm not sure I like this.. I'm not a big fan of global variables... What about introducing these values back into the configuration and provide "setters" or at least documentation that these need to be set manually? That way this can be tested more cleanly and thread-safely without relying on values of global variables (which env is).

Reply:
moved back to configfile